tests/v3/index.test.ts

```diff
@@ -1011,6 +1011,54 @@ describe("OpenAPI3 features", () => {
 
       export interface operations {}
 
+      export interface components {}`)
+    );
+  });
+  it("missing schema in parameteres (#377)", () => {
+    const schema: OpenAPI3 = {
+      openapi: "3.0.1",
+      paths: {
+        "/test/{test_id}": {
+          get: {
+            summary: "some summary",
+            description: "some description",
```

Comment:
you can remove these to simplify the test case. We test these properties in other places